src/cmd/manage_nsfs.js

```diff
@@ -353,25 +344,64 @@ async function account_management(argv, config_root, from_file) {
     await manage_account_operations(action, data, config_root, config_root_backend);
 }
 
+/**
+ * set_access_keys will set the access keys either given as args or generated.
+ * @param {{ access_key: any; secret_key: any; }} argv
+ * @param {boolean} generate a flag for generating the access_keys automatically
+ */
+function set_access_keys(argv, generate) {
+    const { access_key, secret_key } = argv;
+    let generated_access_key;
+    let generated_secret_key;
+    if (generate) {
```

Comment:
so on add account we will always generate creds even though the user provided them? I think we should avoid it if it's not needed

Reply:
Well, that was my initial thought also, but as it is a CLI, and this one is small, and we are not calling it a lot (well manually triggering add), I think it is ok. 
We can always check if argv.access_keys was provided, but as stated in the previous comment, I think we should revoke the option to get access_keys, and only generate/regenerate them.